src/Data/Integer/Properties.agda

```diff
@@ -867,32 +1270,76 @@ n≤1+n n = ≤-step ≤-refl
   ; isStrictTotalOrder = <-isStrictTotalOrder
   }
 
+-- Other properties of _<_
+
 n≮n : ∀ {n} → n ≮ n
 n≮n {+ n}           (+≤+ n<n) =  contradiction n<n ℕₚ.1+n≰n
 n≮n { -[1+ 0 ]}     ()
 n≮n { -[1+ suc n ]} (-≤- n<n) =  contradiction n<n ℕₚ.1+n≰n
 
-<⇒≤ : ∀ {m n} → m < n → m ≤ n
-<⇒≤ m<n =  ≤-trans (n≤1+n _) m<n
+>→≰ : ∀ {x y} → x > y → x ≰ y
```

Comment:
This should be a `⇒` arrow in the name?

Reply:
Then we should also rename [`≰→>`](https://agda.github.io/agda-stdlib/Data.Integer.Properties.html#29357)